compiler/damlc/tests/daml-test-files/SemiLazy.daml

```diff
@@ -0,0 +1,25 @@
+daml 1.2
+module SemiLazy where
```

Comment:
I definitely won’t remember to look for `SemiLazy` when I’m searching for this test. Maybe call it `LazyDo`?

Reply:
That makes a lot of sense. Changed.